src/components/MyKiva/BlogCard.vue

```diff
@@ -77,10 +78,15 @@ const formattedDate = computed(() => {
 </script>
 
 <style scoped>
```

Comment:
Change to `<style lang="postcss" scoped>` so that the `@screen` line doesn't have a warning.